File: c programs/Assignments/Assignment_1/Assign1_5.c
//Accept one number from user and print thet number of * on screen.
#include<stdio.h>

void Accept(int iNo)
{
	int iCnt=0;
	for(iCnt = 0;iCnt<=iNo;iCnt++)
	{
		printf("*");
	}
}


int main()
{
	int iValue=0;
	
	printf("Enter the number");
	scanf("%d",&iValue);
	
	Accept(iValue);
	
	return 0;
}

File: Python/Assignment_1/Assign1_5.py
#Write a program which display 10 to 1 on screen.
#Output : 10 9 8 7 6 5 4 3 2 1 

def Num(iNo):

 iCnt = iNo;
 while(iCnt>0):
   print(iCnt)
   iCnt-= 1


def main():
 iValue=int(input("Enter number"))
 Num(iValue)

if __name__=="__main__":
  main()
